charts/operator-wandb/charts/app/templates/role.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "app.fullname" . }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    {{- include "wandb.commonLabels" . | nindent 4 }}
+    {{- include "app.commonLabels" . | nindent 4 }}
+    {{- include "app.labels" . | nindent 4 }}
+    {{- if .Values.clusterRole.labels -}}
+    {{-   toYaml .Values.clusterRole.labels | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- if .Values.role.annotations -}}
+    {{-   toYaml .Values.role.annotations | nindent 4 }}
+    {{- end }}
+rules:
```

Comment:
We should modify role to include this rule:
  - apiGroups: [""]
    resources: ["namespaces"]
    verbs: ["get"]
We add this to be able to check for existence of (but not create) the namespace

Reply:
why are we not deploying into the namespace the pod is deploy into?

Reply:
We are that role is scoped to the namespace. @andrewtruong can clarify why this is needed.